wp-content/plugins/wporg-learn/inc/post-meta.php

```diff
@@ -51,6 +51,21 @@ function register_lesson_meta() {
 			},
 		),
 	);
+
+	register_post_meta(
+		'lesson',
+		'_lesson_archive_excluded',
+		array(
+			'description'       => __( 'Whether the lesson should be excluded from archive views.', 'wporg-learn' ),
+			'type'              => 'string',
```

Comment:
String based approach follows that of [featured courses](https://github.com/Automattic/sensei/blob/trunk/includes/class-sensei-course.php#L669), and has already been reused for featured lessons.